reference-tests/build.gradle

```diff
@@ -52,6 +58,11 @@ tasks.register('referenceBlockchainTests', Test) {
   description = 'Runs ETH reference blockchain tests.'
   dependsOn generateBlockchainReferenceTests
 
+  boolean isCiServer = System.getenv().containsKey("CI")
+  minHeapSize = isCiServer ? "8g" :"4g"
+  maxHeapSize = isCiServer ? "120g" : "8g"
```

Comment:
I'm not sure if this will work, there is 128GB ram on the runner and if you reserve 120G for jvm, there isn't any left for corset.